src/Controllers/FilesController.js

```diff
@@ -75,24 +75,14 @@ export class FilesController extends AdaptableController {
         // all filenames starting with "tfss-" should be from files.parsetfss.com
         // all filenames starting with a "-" seperated UUID should be from files.parse.com
         // all other filenames have been migrated or created from Parse Server
-        if (config.fileKey === undefined) {
-          fileObject['url'] = this.adapter.getFileLocation(config, filename);
+        if (filename.indexOf('tfss-') === 0) {
+          fileObject['url'] =
+            'http://files.parsetfss.com/' + encodeURIComponent(filename);
+        } else if (legacyFilesRegex.test(filename)) {
+          fileObject['url'] =
+            'http://files.parse.com/' + encodeURIComponent(filename);
         } else {
-          if (filename.indexOf('tfss-') === 0) {
-            fileObject['url'] =
-              'http://files.parsetfss.com/' +
-              config.fileKey +
```

Comment:
Removed "fileKey" from being appended to urls. I assumed legacy parse.com used the fileKey for encryption, didn't realize is was only appended. I made the modification to the legacy test of removing the fileKey from the links there also.

Now "fileKey" is repurposed as an encryption key

Reply:
@dplewis do you have any thoughts on this?

Reply:
I am not sure. I'd prefer to have a new parameter for the encryption as this change will be a breaking change and may affect a lot of apps using it not only changing their files urls but also encrypting their files (and not being able to decrypt the current ones)

Reply:
Sorry for my last comment. I've just checked the code again and fileKey is only being used here in this piece of code. I believe this current code is actually useless now and can be replaced to only
```js
fileObject['url'] = this.adapter.getFileLocation(config, filename);
```

Reply:
I'm only afraid that some developers may have fileKey in their config since parse.com migration and in this case they will have their files being encrypted (maybe without noticing that).

Reply:
Using a new parameter would definitely prevent an accidental usage from people who have the old fileKey set. I think there was an old issue someone opened up where I mentioned for them to remove fileKey from their configuration. I was trying to save an old parameter, but it may not be the best solution here

Reply:
Added and option for `encryptionKey` and used that instead of `fileKey`. Let me know if this works and I can make the change in the docs.

Reply:
I reverted this particular file since I’m no longer using fileKey